src/cookbook/networking/fetch-data.md

```diff
@@ -27,14 +27,12 @@ This recipe uses the following steps:
 The [`http`][] package provides the
 simplest way to fetch data from the internet.
 
-To install the `http` package, add it to the
-dependencies section of the `pubspec.yaml` file.
-You can find the latest version of the
-[`http` package][] the pub.dev.
-
-```yaml
-dependencies:
-  http: <latest_version>
+To install the `http` package,
+add it to the dependencies section of the `pubspec.yaml` file
+by running `flutter pub add`:
```

Comment:
**suggestion:** Write in active voice per [GSG](https://developers.google.com/style/voice).
```suggestion
To install the `http` package, run `flutter pub add`.
This command adds `http` to the `dependencies` section of the `pubspec.yaml` file:
```